test-cypress/integration/ttyg/chat-list.spec.js

```diff
@@ -142,6 +143,27 @@ describe.skip('TTYG chat list', () => {
         // And the chat list should not be visible
         TTYGViewSteps.getChatsPanel().should('be.hidden');
     });
+
+    it('should persist selected chat', () => {
+        TTYGStubs.stubChatsListGet();
+        TTYGStubs.stubChatGet();
+        TTYGStubs.stubAgentListGet();
+        // Given I have opened the ttyg page
+        TTYGViewSteps.visit();
```

Comment:
Put a wait for `cy.wait('@get-chat')` here and after line 162

Reply:
Done